Userland/Libraries/LibWeb/IndexedDB/IDBRequest.h

```diff
@@ -27,6 +37,20 @@ class IDBRequest : public DOM::EventTarget {
     explicit IDBRequest(JS::Realm&);
 
     virtual void initialize(JS::Realm&) override;
+    virtual void visit_edges(Visitor& visitor) override;
+
+private:
+    // A request has a processed flag which is initially false.
+    bool m_processed { false };
+    // A request has a done flag which is initially false.
+    bool m_done { false };
+    // A request has a result and an error
+    JS::Value m_result;
+    JS::GCPtr<WebIDL::DOMException> m_error;
+    // FIXME: A request has a source object.
+    // IDBRequestSource m_source;
+    // FIXME: A request has a transaction which is initially null.
+    // JS::GCPtr<IDBTransaction> m_transaction;
```

Comment:
:T leaving these out before adding them later would be preferable to having them as commented out members